build.gradle

```diff
@@ -101,6 +101,7 @@ allprojects {
       "--add-opens=java.base/java.lang=ALL-UNNAMED",
       "--add-opens=java.base/java.util=ALL-UNNAMED"
     ]
+    maxHeapSize = "2g"
```

Comment:
Hopefully we will not forget mentioning this in the release notes.

Reply:
There would be no need; this is a build configuration that ensures the tests have sufficient memory to run and will not affect the distributed artifact in any way.